cg/meta/workflow/mutant/quality_controller/utils.py

```diff
@@ -0,0 +1,104 @@
+from cg.apps.lims.api import LimsAPI
+from cg.constants.constants import MutantQC
+from cg.constants.lims import LimsArtifactTypes, LimsProcess
+from cg.exc import CgError, LimsDataError
+from cg.meta.workflow.mutant.metrics_parser.metrics_parser import MetricsParser
+from cg.meta.workflow.mutant.metrics_parser.models import SampleResults
+from cg.meta.workflow.mutant.quality_controller.models import (
+    MutantPoolSamples,
+    QualityMetrics,
+)
+from genologics.entities import Artifact
+from genologics.entities import Sample as LimsSample
+from cg.models.cg_config import LOG
+from cg.services.sequencing_qc_service.quality_checks.utils import sample_has_enough_reads
+from cg.store.models import Case, Sample
+
+from pathlib import Path
+
+from cg.store.store import Store
+
+
+def has_valid_total_reads(
+    sample: Sample,
+    external_negative_control: bool = False,
+    internal_negative_control: bool = False,
+) -> bool:
+    if external_negative_control:
+        return external_negative_control_sample_has_enough_reads(reads=sample.reads)
+
+    if internal_negative_control:
+        return internal_negative_control_sample_has_enough_reads(reads=sample.reads)
+
+    return sample_has_enough_reads(sample=sample)
+
+
+def external_negative_control_sample_has_enough_reads(reads: int) -> bool:
+    return reads < MutantQC.EXTERNAL_NEGATIVE_CONTROL_READS_THRESHOLD
+
+
+def internal_negative_control_sample_has_enough_reads(reads: int) -> bool:
+    return reads < MutantQC.INTERNAL_NEGATIVE_CONTROL_READS_THRESHOLD
+
+
+def get_internal_negative_control_id_for_case(lims: LimsAPI, case: Case) -> str:
```

Comment:
@islean and @ChrOertlin 
Here I'm looking for a function which returns the negative control sample for a case (a mutant pool of samples)!

Reply:
Understood! Could still make sense to wrap a function taking the sample_internal_id as input, i.e. get_internal_negative_control_id_for_case(case) calls a get_internal_negative_control_id(sample_internal_id). Up to you, though.

Reply:
So this `get_internal_negative_control_id_for_case` function calls the `get_internal_negative_control_id_from_sample_in_pool`. Do you think another wrapper is necessary?

Reply:
Resolving this as the code was moved.